src/prefabs/StatusIcon.tsx

```diff
@@ -42,9 +42,11 @@ const IconTimer: React.FC<IconTimerProps> = ({ time, width, height }) => {
     const text = useMemo(() => {
         if (time < 60) {
             return time.toString();
+        } else if (time < 3600) {
+            return `${Math.floor(time / 60)}m`;
```

Comment:
The previous `if` block unconditionally returns, so `else` is redundant.
```suggestion
        }
        if (time < 3600) {
            return `${Math.floor(time / 60)}m`;
```